doc/docs.md

```diff
@@ -710,21 +710,30 @@ struct Point {
     y int
 }
 
-p := Point{
+mut p := Point{
     x: 10
     y: 20
 }
 
 println(p.x) // Struct fields are accessed using a dot
+
+// Alternative literal syntax for structs with 3 fields or fewer
+p = Point{10, 20}
+assert p.x == 10
+
+// you can omit the struct name when it's already known
+p = {x: 30, y: 4}
+assert p.y == 4
 ```
 
+Omitting the struct name also works for function arguments.
```

Comment:
This is covered in https://github.com/vlang/v/blob/master/doc/docs.md#short-struct-initialization-syntax

I don't think it needs to be duplicated.

Reply:
Actually that section doesn't mention `func({struct_field: init})`.
I've just added a commit here that tweaks that section.

Reply:
I decided it would be better to put all the short syntax code in one section - see #5762.